### ISSUE TITLE
Bindkey for editor is overwritten?

### ISSUE THREAD

**Issue by smcallis** [leophys/zsh-plugin-fzf-finder#3]
Hey I think these lines:

```
zle -N fzf-finder-widget-editor
bindkey ${FZF_FINDER_BINDKEY:-'\ee'} fzf-finder-widget-editor
zle -N fzf-finder-widget-pager
bindkey ${FZF_FINDER_BINDKEY:-'\er'} fzf-finder-widget-pager
```
Have the effect of just overwriting FZF_FINDER_BINDKEY if it's set by the user, so you always get the pager, not the editor.

Easy workaround, just FYI.

**Comment by leophys** [comment]
Ehi, you're right. Thanks!

I'll fix it shortly.

**Comment by leophys** [comment]
Closed by 90322c8
